airbyte-integrations/connectors/source-amazon-ads/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.5
+LABEL io.airbyte.version=0.1.6
```

Comment:
@tirth7777777 please check your commit because is changing Amazon Ads too.

Reply:
oh sorry. I will correct that ASAP